<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmWareForm</class>
 <widget class="QWidget" name="FirmWareForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>108</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>烧写进度：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtShow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>145</y>
     <width>701</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSelFile">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>49</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择固件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFlashing">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>49</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始烧写</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="proBar">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>105</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>55</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbBoxBuad">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>53</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbBoxPort">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>52</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>55</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>烧写串口：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
